lib/ansible/module_utils/facts/system/pkg_mgr.py

```diff
@@ -73,21 +73,24 @@ class PkgMgrFactCollector(BaseFactCollector):
 
     def _check_rh_versions(self, pkg_mgr_name, collected_facts):
         if collected_facts['ansible_distribution'] == 'Fedora':
-            try:
-                if int(collected_facts['ansible_distribution_major_version']) < 23:
-                    for yum in [pkg_mgr for pkg_mgr in PKG_MGRS if pkg_mgr['name'] == 'yum']:
-                        if os.path.exists(yum['path']):
-                            pkg_mgr_name = 'yum'
-                            break
-                else:
-                    for dnf in [pkg_mgr for pkg_mgr in PKG_MGRS if pkg_mgr['name'] == 'dnf']:
-                        if os.path.exists(dnf['path']):
-                            pkg_mgr_name = 'dnf'
-                            break
-            except ValueError:
-                # If there's some new magical Fedora version in the future,
-                # just default to dnf
-                pkg_mgr_name = 'dnf'
+            if os.path.exists('/run/ostree-booted'):
+                return "atomic_container"
+            else:
```

Comment:
There is no need for the `else:` and indentation since the previous block returns.

Will make for a better diff as well.

Reply:
fair point 👍